src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -70,6 +70,29 @@
                 *ngIf="showSubmit">Are you sure you want to continue?</ng-container>
 </ng-template>
 
+
+<ng-template #hostMetricTmpl
+             let-value="value">
+  <div *ngIf="validValue(value)">
+    <span>{{ value }}</span>
+  </div>
+  <div *ngIf="!validValue(value)">
+    <span class="badge-warning"
+          ngbTooltip="Not available. Data could not be fetched from Ceph">N/A</span>
+  </div>
+</ng-template>
+
+<ng-template #hostDimlessTmpl
+             let-value="value">
+  <div *ngIf="!validValue(value)">
+    <span class="badge-warning"
```

Comment:
This is not an error, is a warning. I think not receiving data that should be available is not only "info" it is probably something to be taken into account since there might be something wrong.

Reply:
yup, that's true! one other concern i have is with a cephadm cluster when we add a host it doesn't have this info yet because the host is getting ready. and after a while it'll fetch those info's. So wouldn't it be a noise to show these warnings with `Data could not be fetched from Ceph` message?